packages/e2e-tests/specs/experiments/__snapshots__/navigation-editor.test.js.snap

```diff
@@ -2,11 +2,7 @@
 
 exports[`Navigation editor allows creation of a menu 1`] = `
 "<!-- wp:navigation -->
-<!-- wp:navigation-link {\\"label\\":\\"Home\\",\\"type\\":\\"page\\",\\"id\\":1,\\"url\\":\\"https://this/is/a/test/page/home\\"} /-->
-
-<!-- wp:navigation-link {\\"label\\":\\"About\\",\\"type\\":\\"page\\",\\"id\\":2,\\"url\\":\\"https://this/is/a/test/page/about\\"} /-->
-
-<!-- wp:navigation-link {\\"label\\":\\"Contact\\",\\"type\\":\\"page\\",\\"id\\":3,\\"url\\":\\"https://this/is/a/test/page/contact\\"} /-->
+<!-- wp:page-list /-->
```

Comment:
Hmm, this test doesn't really assert much now that the snapshot doesn't show the menu structure. We don't know if page-list is actually working. I wonder what else we can do here.

Reply:
How about we do something like `page.$$('.page-item')` and check that it returns the expected number of items?

Reply:
Yeah, I think that'd be good 👍